alerts.php

```diff
@@ -27,33 +27,34 @@
  * @author: Heath Barnhart <hbarnhart@kanren.net>
  */
 
-use LibreNMS\Util\FileLock;
 use LibreNMS\Alert\RunAlerts;
+use LibreNMS\Util\FileLock;
```

Comment:
Typo?

Reply:
Legacy, I was looking at refactoring the existing locks, but went this route instead.  Thanks, removed this and one other reference.